File: 프로그래머스/lv0/120893. 대문자와 소문자/대문자와 소문자.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string my_string)
{
    for (int i = 0; i < my_string.length(); i++)
    {
        if (my_string[i] >= 'a' && my_string[i] <= 'z')
            my_string[i] -= 32;
        else if (my_string[i] >= 'A' && my_string[i] <= 'Z')
            my_string[i] += 32;
    }
    return my_string;
}

File: 프로그래머스/0/120893. 대문자와 소문자/대문자와 소문자.java
class Solution {
    public String solution(String my_string) {
        StringBuilder answer = new StringBuilder();
        
        for (char c : my_string.toCharArray()) {
            if (Character.isUpperCase(c)) {
                answer.append(Character.toLowerCase(c));
            } else {
                answer.append(Character.toUpperCase(c));
            }
        }
        
        return answer.toString();
    }
}